docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md

```diff
@@ -61,19 +61,19 @@ If you enable remote debugging for a cloud service, it doesn't exhibit degraded
 
 ### To attach the debugger to a cloud service in Azure
 
-1. In Server Explorer, expand the node for your cloud service.
+1. In **Server Explorer**, expand the node for your cloud service.
```

Comment:
This should remain plain text. Applications and their components (windows, explorers) should be plain text unless you are describing an action like "select **Server Explorer**, and then select . . ."

Reply:
hi, @ktoliver , sorry to check with you again. I reverted all the bold face I added on "server explorer". However, I noticed that some contexts like this one specifically are in interactions with the UI. Even they are not selected but it's the interface of Server Explorer where users interact. Please see many original occurrences in block#1 files. For example, **Cloud Explorer** in docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md.
I'd think such alignment of using bold face for UI elements need your discussion with your team before I make further changes.